usr/lib/python3/dist-packages/linuxmusterLinuxclient7/printers.py

```diff
@@ -73,16 +73,20 @@ def translateSambaToIpp(networkPath):
 # --------------------
 
 def _installPrinter(username, networkPath, name):
-    logging.debug("Installing Printer {0} on {1}".format(name, networkPath))
-    installCommand = ["lpadmin", "-p", name, "-E", "-v", networkPath, "-m", "everywhere", "-u", f"allow:{username}"]
+    logging.info("_installPrinter {0} on {1}".format(name, networkPath))
```

Comment:
Why did you change that?

Reply:
wegen Konsistenz zu anderen Funktionsaufrufen, die auch .info verwenden

Reply:
Could you please provide an example? I cannot find an occurrence of something like this anywhere else.
Furthermore, `_installPrinter` is an internal function (indicated by the `_`) and should therefore not be mentioned in logs, they are meant for users, not for developers.